PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalCheckoutRequest.kt

```diff
@@ -64,6 +64,7 @@ class PayPalCheckoutRequest @JvmOverloads constructor(
     var currencyCode: String? = null,
     var shouldRequestBillingAgreement: Boolean = false,
     var shouldOfferPayLater: Boolean = false,
+    var contactInformation: PayPalContactInformation? = null,
```

Comment:
I know this is the final PR not requiring review but realized we are missing the doc string for this property - iOS has `Optional: Contact information of the recipient for the order`

Reply:
Good catch, looks like we don't use the `Optional` part on Android based on the docstrings here, added the missing one here: https://github.com/braintree/braintree_android/pull/1250/commits/028271a5d50fc0132a6abea2222eb362465f000e